Decred Wallet/Features/Security/RequestPinViewController.swift

```diff
@@ -17,95 +17,112 @@ class RequestPinViewController: SecurityBaseViewController {
     var onUserEnteredPin: ((_ pin: String) -> Void)?
     
     var requestPinConfirmation = false
-    var pinToConfirm: String = ""
     
-    @IBOutlet weak var cancelBtn: UIButton!
+    var submitEnabled: Bool = false{
```

Comment:
Check your opening and closing braces here and add whitespaces as appropriate.

Reply:
resolved in 50488de